code/game/turfs/open/openspace.dm

```diff
@@ -33,6 +33,10 @@ GLOBAL_DATUM_INIT(openspace_backdrop_one_for_all, /atom/movable/openspace_backdr
 	neighborlay_override = "staticedge"
 	turf_flags = NONE
 
+/turf/open/transparent/openspace/Initialize()
+	. = ..()
+	icon_state = "transparent"
```

Comment:
adding init bloat one pr at a time, please revert this, their is so many openspace turfs that this will have notable impact on init

Reply:
Will revert it but, Is there a way of adding a map only sprite otherwise? since the invisible tiles make it really easy to make mistakes

Reply:
unfortunately not that I'm aware of